DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml

```diff
@@ -31,52 +34,8 @@
   </a>
 </div>
 
-<div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
-  <div class="nhsuk-card__content">
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-three-quarters">
-        <h3 class="nhsuk-card__heading">
-          Diagnostic Assessment
-        </h3>
-      </div>
-      <div class="nhsuk-grid-column-one-quarter">
-        <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color">
-          Not attempted
-        </h3>
-      </div>
-    </div>
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-full">
-        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0">
-          Launch assessment
-        </a>
-      </div>
-    </div>
-  </div>
-</div>
+<partial name="_DiagnosticAssessmentCard" model="diagnosticAssessmentStatus" />
```

Comment:
No need to pass in any model here as everything is hardcoded for now. Once we have the data from the backend we'll be passing in the real model 🙂

Reply:
Ah my bad, that's from my original approach of using the same partial view.